src/hazelcore/util/StringUtil.re

```diff
@@ -54,3 +54,16 @@ let num_trailing_zeros = (num: string): int => {
     };
   helper(String.length(num) - 1);
 };
+
+let strip_underscores = (s: string): string => {
+  let rec helper = (index: int, stripped: string): string =>
```

Comment:
There is likely a more efficient way using standard library functions to drop a single character.